SCClassLibrary/Common/GUI/tools/Stethoscope.sc

```diff
@@ -437,8 +437,9 @@ Stethoscope {
 		);
 		^true;
 	}
-	
+
 	bounds_ { arg rect;
+		rect = rect ?? view.bounds;
```

Comment:
should be 
`rect = rect ?? { view.bounds };`

Reply:
Thanks! Done!